pkg/controllers/netpol/network_policy_controller.go

```diff
@@ -503,7 +503,7 @@ func (npc *NetworkPolicyController) ensureExplicitAccept() {
 		for _, chain := range defaultChains {
 			comment := "\"rule to explicitly ACCEPT traffic that comply to network policies\""
```

Comment:
Modify this as well to reflect that it is jumping to RETURN not ACCEPT.

Reply:
I am only updating the branch with the commit from upstream. I have another commit to fix the issue with an updated comment I'll create a new PR after this is merged.

Reply:
Ok! Was not quite sure whats going on, if a  branch on our fork needs to be fast-forwarded to track with upstream I would usually just push to it directly instead of opening a PR.